src/LuaTransformer.ts

```diff
@@ -2972,7 +2972,15 @@ export class LuaTransformer {
             hasContext ? context : undefined
         );
 
-        const body = ts.isBlock(node.body) ? node.body : ts.createBlock([ts.createReturn(node.body)]);
+        let body: ts.Block;
+        if (ts.isBlock(node.body)) {
+            body = node.body;
+        } else {
+            const ret = ts.createReturn(node.body);
```

Comment:
This could use a better variable name 😄

Reply:
Oops - I had meant to go back and rename that before 😄